src/api/v1/ssids.js

```diff
@@ -0,0 +1,30 @@
+const config = require('../../config')
+
+let ssidsIndexUrl = `${config.apiBaseUrl}/ssids`
+
+module.exports = {
+  count: function() {
+	return fetch(ssidsIndexUrl)
+	  .then(response => response.json())
+	  .then(data => {
+      document.getElementById('ssids-count').textContent = data.length
```

Comment:
I need to add an SSID count end point to the API.  This is going to get kind of slow for large data sets.  It will do for now, though.

Reply:
I thought similarly, will update if a new endpoint is merged.